chatapi_ros/README.md

```diff
@@ -0,0 +1,90 @@
+# chatapi_ros
+The purpose of this repository is to make existing chat APIs usable with ROS.
+
+## Setting up environment
+Please install following packages.
+```
+sudo pip install google-api-python-client
+sudo pip install oauth2client
+sudo pip install pya3rt
```

Comment:
pip sometime cause troubles (https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/248), so if this is just send POST response, use `requests.post` directly like chaplus would be good option
https://github.com/nocotan/pya3rt/blob/master/pya3rt/talk.py